src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateRenderField.tsx

```diff
@@ -102,8 +133,22 @@ export function RenderedField({
     return selectedDisplayMode?.name;
   };
 
+  const getFromResponseRenderedValue = (value: unknown) => {
```

Comment:
Same comment as earlier regarding identifier naming consistency:
```suggestion
  const getResponseFromRenderedValue = (value: unknown) => {
```